cartographer_ros/cartographer_ros/node_main.cc

```diff
@@ -28,6 +28,9 @@ DEFINE_string(configuration_basename, "",
               "Basename, i.e. not containing any directory prefix, of the "
               "configuration file.");
 DEFINE_string(map_filename, "", "If non-empty, filename of a map to load.");
+DEFINE_bool(
+    start_trajectory_with_default_topics, true,
+    "Enable/disable to start the first trajectory with default topics.");
```

Comment:
For consistency: "Enable to immediately start the first trajectory with default topics."